README.md

```diff
@@ -26,11 +26,11 @@ More about Cake at [cakebuild.net](http://cakebuild.net).
 
 - [Documentation](https://cake-contrib.github.io/Cake.Npm)
 
-## Chat Room
+## Discussion
 
-Come join in the conversation about Cake.Yeoman in our Gitter Chat Room.
+If you have questions, search for an existing one, or create a new discussion on the Cake GitHub repository, using the `extension-q-a` category.
 
-[![Join the chat at https://gitter.im/cake-contrib/Lobby](https://badges.gitter.im/cake-contrib/Lobby.svg)](https://gitter.im/cake-contrib/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Join in the discussion on the Cake repository](https://img.shields.io/badge/GitHub-Discussions-green?logo=github)](https://github.com/cake-build/cake/discussions)
```

Comment:
Link should go to the correct category: https://github.com/cake-build/cake/discussions/categories/extension-q-a